libs/sdk-ui-kit/api/sdk-ui-kit.api.md

```diff
@@ -517,6 +517,8 @@ export interface IAppHeaderProps {
     // (undocumented)
     helpRedirectUrl?: string;
     // (undocumented)
+    isDisplayUpsellButton?: boolean;
```

Comment:
should we rename `isDisplayUpsellButton` → `shouldDisplayUpsellButton` to be more grammatically correct?

Reply:
I renamed it thanks